### PR TITLE
Update attack-surface-reduction-rules-deployment.md

### LINKED ISSUE #12414: Remove old text
You can remove this line: 

(1) Block abuse of exploited vulnerable signed drivers isn't currently available in Intune Endpoint security. You can configure this rule using [Intune OMA-URI](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/enable-attack-surface-reduction?view=o365-worldwide#custom-profile-in-intune).

Block abuse of exploited vulnerable signed drivers  - is now available under Endpoint Security > Attack Surface Reduction 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 8be094e8-a6e3-a995-0916-9d8a29006d32
* Version Independent ID: 8be094e8-a6e3-a995-0916-9d8a29006d32
* Content: [Microsoft Defender for Endpoint (MDE) attack surface reduction (ASR) rules deployment overview](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
added statement about  Block abuse of exploited vulnerable signed drivers as it is currently available

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12414

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ As outlined in [Use attack surface reduction rules to prevent malware infection]
 | Use advanced protection against ransomware | Block persistence through WMI event subscription | Block Office apps from injecting code into other processes | Block Office communication apps from creating child processes | | |
 | | | Block Adobe Reader from creating child processes | | | |
 
-(<a id="fn1">1</a>) _Block abuse of exploited vulnerable signed drivers_ isn't currently available in Intune Endpoint security. You can configure this rule using [Intune OMA-URI](enable-attack-surface-reduction.md#custom-profile-in-intune).
+(<a id="fn1">1</a>) _Block abuse of exploited vulnerable signed drivers_ is now available under **Endpoint Security** > **Attack Surface Reduction**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-deployment.md]:
```suggestion
(<a id="fn1">1</a>) _Block abuse of exploited vulnerable signed drivers_ is now available under **Endpoint Security** > **Attack Surface Reduction**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba

**Pull request merged**